main.go

```diff
@@ -2,41 +2,58 @@ package main
 
 import (
 	"database/sql"
+	"flag"
 	"fmt"
 	"net/http"
 	"os"
+	"strconv"
+	"strings"
 	"time"
 
 	_ "github.com/mattn/go-sqlite3"
 	"github.com/op/go-logging"
 )
 
+const (
+	portDefault = "8080"
+)
+
 var logger = logging.MustGetLogger("commento")
 var db *sql.DB
 
+// The run environment for the application. Either dev, staging, or prod
+var runEnv string
+
 func main() {
-	err := loadDatabase("sqlite3.db")
-	if err != nil {
+	if err := loadDatabase("sqlite3.db"); err != nil {
 		die(err)
 	}
 
-	fs := http.FileServer(http.Dir("assets"))
+	// Parse command line options
+	var port = flag.String("port", portDefault, "port for commento service")
+	flag.Parse()
 
-	http.Handle("/assets/", http.StripPrefix("/assets/", fs))
-	http.HandleFunc("/", indexHandler)
-	http.HandleFunc("/create", createCommentHandler)
-	http.HandleFunc("/get", getCommentsHandler)
-
-	var port string
+        // Check that we get a valid port value
+	if _, err := strconv.ParseInt(*port, 10, 32); err != nil {
```

Comment:
I could have parsed the port as an Int but it would have to translated into a string to be used with ListenAndServe. We would be trading the check made here for a conversation made elsewhere in other words.

Also the port is never really used as an integer (no mathematical operations or storage as such). It's only function is as a string passed to ListenAndServe.